common/common.go

```diff
@@ -5,8 +5,9 @@ type TestConfig struct {
 }
 
 func Run(cfg *TestConfig) {
-	runBasic()
 	runApps()
+	runBasic()
```

Comment:
I hope the order here’s isn’t important. It’ll not be guaranteed once parallel tests are implemented.

Reply:
I just ordered it alphabetically.